CalibTracker/SiStripLorentzAngle/interface/TIBorTOBparameters.h

```diff
@@ -0,0 +1,21 @@
+#ifndef TIBORTOBPARAMETERS_H
+#define TIBORTOBPARAMETERS_H
+
+#include "DataFormats/SiStripDetId/interface/SiStripDetId.h"
+
+// WARNING: This header file is created on the assumption
+// that the subDetector Id (TIB or TOB) is already known.
+// It is used inside CalibTracker/SiStripLorentzAngle only.
```

Comment:
How about something like "WARNING: these methods assume that the DetId is a TIBDetId or TOBDetId (and don't check this). Please use the corresponding TrackerTopology methods whenever possible.
Code copied from TIBDetId and TOBDetId.